styleguide/public/sassdocs/assets/js/main.js

```diff
@@ -0,0 +1,56 @@
+/* global document */
+
+(function ($, global) {
```

Comment:
No Jquery please.
I don't see it in de sassdoc dependencies, nor in our own package.json.